src/vehicle.h

```diff
@@ -708,6 +708,9 @@ class vehicle : public JsonSerializer, public JsonDeserializer
     // <0 means there is no steering installed at all.
     float steering_effectiveness() const;
 
+    /** Returns roughly driving skill level at which there is no chance of fumbling. */
+    float handling_difficulty() const;
```

Comment:
Roughly because the skill level is affected by other factors. Currently just dexterity and perception, but later could be speed, stimulant level (both positive and negative could be a penalty, as in real life) etc. Meaning it isn't exactly just the driving skill level, even though it's the most important factor.

Not sure how to word it better.